src/pretix/plugins/sendmail/signals.py

```diff
@@ -1,9 +1,26 @@
+from django.db.models.signals import post_save
 from django.dispatch import receiver
 from django.urls import resolve, reverse
 from django.utils.translation import gettext_lazy as _
 
+from pretix.base.models import SubEvent
 from pretix.base.signals import logentry_display
 from pretix.control.signals import nav_event
+from pretix.plugins.sendmail.models import ScheduledMail
+
+
+@receiver(post_save, sender=SubEvent)
+def sendmail(sender, **kwargs):
+
+    subevent = kwargs.get('instance')
+    event = subevent.event
+    rules = event.rule_set
+    qs = ScheduledMail.objects.filter(subevent=subevent)
+
+    if not qs.exists():
+        for rule in rules.all():
+            if not qs.filter(rule=rule).exists():
```

Comment:
if we have ``n`` rules and create a new subevent, this will cause `2n+2` database queries. And I'm not even sure if the second ``if`` statement could ever be true if the first one is.

An alternative could be:

```
existing_rules = ScheduledMail.objects.filter(subevent=subevent).values_list('rule_id', flat=True)
to_create = []
for rule in event.rule_set.all():
    if rule.pk not in existing_rules:
        to_create.append(ScheduledMail(rule=rule, event=event, subevent=subevent))
ScheduledMail.objects.bulk_create(to_create)
```

This would get us down to a constant 3 queries per subevent, regardless of the number of rules.